apps/gitness/src/pages/repo/repo-list.tsx

```diff
@@ -37,17 +38,16 @@ export default function ReposListPage() {
       data && (
         <RepoList
           LinkComponent={LinkComponent}
-          // @ts-expect-error remove "@ts-expect-error" once type issue for "content" is resolved
-          repos={data?.content?.map(repo => {
+          repos={data?.map((repo: RepoRepositoryOutput) => {
             return {
               id: repo.id,
               name: repo.identifier,
               description: repo.description,
-              private: repo.private,
-              stars: repo.stars,
-              forks: repo.forks,
-              pulls: repo.pulls,
-              timestamp: repo.timestamp
+              private: repo.is_public,
+              stars: 0,
```

Comment:
hardcoded 0?

Reply:
Removed it, also made `stars` attribute optional for now. Currently, repo listing api does not have `star` count in it. I  have also added it to the API requirement doc.